RecoTracker/TkHitPairs/interface/CosmicLayerPairs.h

```diff
@@ -15,22 +15,28 @@
 #include <vector>
 
 class TrackerTopology;
+class GeometricSearchTracker;
 
 class CosmicLayerPairs : public SeedLayerPairs {
 public:
-  CosmicLayerPairs(std::string geometry) : _geometry(geometry){};  //:isFirstCall(true){};
+  CosmicLayerPairs(std::string geometry,
+                   const SiStripRecHit2DCollection &collrphi,
+                   const SiStripMatchedRecHit2DCollection &collmatched,
+                   const GeometricSearchTracker &track,
+                   const TrackerTopology &ttopo)
+      : _geometry(geometry) {
+    init(collrphi, collmatched, track, ttopo);
+  };  //:isFirstCall(true){};
```

Comment:
```suggestion
  };
```
this seems misplaced here now.
Also, it may be better to default-initialize at L40
... the other alternative is `_geometry(geometry) { //,sFirstCall(true){`

Reply:
removed as variable is no longer needed anywhere